### DISCUSSION TITLE
BH1750 Light sensor

### DISCUSSION THREAD

**Discussion by gfn256** [arendst/Tasmota#17061] [Q&A]
### PROBLEM DESCRIPTION
_A clear and concise description of what the problem is._
Does Tasmota - Allsensors (from tasmota web installer - development) support the BH1750 light sensor
I need this one because I also want it to support the IKEA VINDRIKTNING 
I believe this is the link to the firmware specials:
https://github.com/Jason2866/Tasmota-specials/blob/firmware/firmware/tasmota/other/tasmota-allsensors.bin 

Thanks

### REQUESTED INFORMATION
_Make sure your have performed every step and checked the applicable boxes before submitting your issue. Thank you!_

- [x] Read the [Contributing Guide and Policy](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md) and [the Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
- [x] Searched the problem in [issues](https://github.com/arendst/Tasmota/issues)
- [x] Searched the problem in [discussions](https://github.com/arendst/Tasmota/discussions)
- [x] Searched the problem in the [docs](https://tasmota.github.io/docs/FAQ)
- [x] Searched the problem in the [chat](https://discord.gg/Ks2Kzd4)
- [x] Device used (e.g., Sonoff Basic): _____D1 MINI PRO (esp8266ex)
- [x] Tasmota binary firmware version number used: _____12.2.0.3
  - [x] Pre-compiled
  - [X] Self-compiled
- [x] Flashing tools used: _____
- [x] Provide the output of command: `Backlog Template; Module; GPIO 255`:
```lua
  Configuration output here:


```
- [x] If using rules, provide the output of this command: `Backlog Rule1; Rule2; Rule3`:
```lua
  Rules output here:


```
- [x] Provide the output of this command: `Status 0`:
```lua
  STATUS 0 output here:


```
- [x] Set `weblog` to 4 and then, when you experience your issue, provide the output of the Console log:
```lua
  Console output here:


```

### TO REPRODUCE
_Steps to reproduce the behavior:_

### EXPECTED BEHAVIOUR
_A clear and concise description of what you expected to happen._

### SCREENSHOTS
_If applicable, add screenshots to help explain your problem._

### ADDITIONAL CONTEXT
_Add any other context about the problem here._

**(Please, remember to close the issue when the problem has been addressed)**

**Comment by heligan** [comment]
Tasmota does support BH1750 via i2c.

https://tasmota.github.io/docs/BH1750/

See a screenshot from my recent post

<img width="367" alt="Screen Shot 2022-11-04 at 9 36 18 PM" src="https://user-images.githubusercontent.com/5983507/200438238-6328e71d-d70b-41df-8768-b776445aaf02.png">

You can check your build with `I2cDriver` (running this command in tasmota web console) to list all i2c drivers linked into your firmware. BH1750 driver should be identified as `11`, check this page

https://tasmota.github.io/docs/I2CDEVICES/

 Usually it detects the sensor automatically if your wiring / configuration is correct and a proper driver is present.

**Comment by vporsio** [comment]
Same issue for me.
In the Allsensors bin there is another drivers for 0x23 address. You need to disable driver 2
